libzmix/src/zkl/mod.rs

```diff
@@ -3,7 +3,7 @@ pub mod spec;
 pub struct Parser {}
 
 impl Parser {
-    pub fn parse(proof_spec: &spec::ProofSpec, witness: &Witness) -> Result<Proof, ZKLError> {
+    pub fn parse(_proof_spec: &spec::ProofSpec, _witness: &Witness) -> Result<Proof, ZKLError> {
```

Comment:
Why not use `#[allow(unused_variables)]`. The reason compiler is complaining about the name is that the function is not implemented. What if we had a Parser trait and this was just one of the implementation. We would not want to change arg names.

Reply:
I agree with this. Making the changes.